src/worknet-ext/logger/WorkNetLogger.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.Extensions.Configuration;
+using Neo;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Plugins;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+
+namespace WorkNetExt;
+
+public class WorkNetLogger : Plugin
+{
+    private string _logFile = "./logger.log";
+
+    NeoSystem? neoSystem;
+
+    public WorkNetLogger()
+    {
+        Blockchain.Committing += OnCommitting;
+        ApplicationEngine.Log += OnAppEngineLog!;
+        Neo.Utility.Logging += OnNeoUtilityLog;
+
+    }
+
+    public override void Dispose()
+    {
+        Neo.Utility.Logging -= OnNeoUtilityLog;
+        ApplicationEngine.Log -= OnAppEngineLog!;
+        Blockchain.Committing -= OnCommitting;
+        GC.SuppressFinalize(this);
+    }
+
+    // Overwrite Config file method to find the config.json from the same directory as the plugin dll directory
```

Comment:
Please expand this comment so that plug in developers undertand that overriding ConfigFile is an absolute requirement for building plugins that are worknet compatible